impl/testdata/set_tasks_with_then.yaml

```diff
@@ -0,0 +1,30 @@
+# Copyright 2025 The Serverless Workflow Specification Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+document:
+  name: then-workflow
+  dsl: '1.0.0-alpha5'
+  namespace: default
+  version: '1.0.0'
+do:
+  - task1:
+      set:
+        value: 30
+      then: task3
+  - task2:
+      set:
+        skipped: true
```

Comment:
even if this isn't skipped, task1 will jump directly to task3, right?
Shouldn't task three have the end statement?

Reply:
There's no `end` in the new spec. It's assumed based on tasks order.

Reply:
interesting.